pyuvdata/uvdata/tests/test_mir.py

```diff
@@ -0,0 +1,107 @@
+# -*- mode: python; coding: utf-8 -*-
+# Copyright (c) 2018 Radio Astronomy Software Group
+# Licensed under the 2-clause BSD License
+
+"""Tests for Mir object.
+
+"""
+import os
+
+import pytest
+import numpy as np
+
+from pyuvdata import UVData
+from pyuvdata.data import DATA_PATH
+from pyuvdata.uvdata.mir import mir_parser
+
+
+@pytest.fixture
+def uv_in_uvfits(tmp_path):
+    uv_in = UVData()
+    testfile = os.path.join(DATA_PATH, "sma_test.mir")
+    write_file = str(tmp_path / "outtest_mir.uvfits")
+
+    # Currently only one source and one spectral window are supported.
+    uv_in.read(testfile)
+    uv_out = UVData()
+
+    yield uv_in, uv_out, write_file
+
+    # cleanup
+    del uv_in, uv_out
+
+
+def test_read_mir_write_uvfits(uv_in_uvfits):
+    """
+    Mir to uvfits loopback test.
+
+    Read in Mir files, write out as uvfits, read back in and check for
+    object equality.
+    """
+    mir_uv, uvfits_uv, testfile = uv_in_uvfits
+
+    mir_uv.write_uvfits(testfile, spoof_nonessential=True)
+    uvfits_uv.read_uvfits(testfile)
+
+    # test fails because of updated history, so this is our workaround for now.
```

Comment:
pushed in f53e2e83

Reply:
Had to push an additional fix for this -- since pyuvdata seems to always append the version number to the file history on write, there wasn't an immediately clear way to use `_check_histories` to accomplish the above goal. As a workaround, I used the function `issubset` to verify that at least the first part of the file histories match up, which seems to work well.  Pushed in ed9b703.